tests/src/test/scala/cats/tests/ListWrapper.scala

```diff
@@ -61,7 +60,13 @@ object ListWrapper {
 
   val functor: Functor[ListWrapper] = traverse
 
-  val invariant: Invariant[ListWrapper] = functor
+  val invariant: InvariantSemigroupal[ListWrapper] = new InvariantSemigroupal[ListWrapper] {
```

Comment:
I have the same comment here that I previously left on the `traverse` instance. There may be tests that are checking implicit resolution when `invariant` is not returning anything more specific than `Invariant`. If we need to, we can add a separate method that returns an `InvariantSemigroupal[ListWrapper]`.